testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -680,3 +680,30 @@ def forward(self, x):
 input = Variable(torch.randn(seq_len, batch, features))
 lstm = LSTM(features, hidden, batch, bidirectional=True)
 save_data_and_model("lstm_bidirectional", input, lstm)
+
+from torchvision.ops.misc import *
+n = 64
+model = FrozenBatchNorm2d(n)
+model.eval()
+input = Variable(torch.rand( 1, 64, 1, 1 ))
+save_data_and_model("frozenBatchNorm2d", input, model)
+
+import torch.nn.functional as F
+class UpsampleUnfusedTwoInput(nn.Module):
+
+    def __init__(self):
+        super(UpsampleUnfusedTwoInput, self).__init__()
+        self.conv1 = nn.Conv2d(3, 3, kernel_size=1, stride=1, padding=0)
+        self.conv2 = nn.Conv2d(3, 3, kernel_size=1, stride=1, padding=0)
+
+    def forward(self, x, y):
+        x = self.conv1(x)
+        x = x.shape[-2:]
+        y = self.conv2(y)
+        y = F.interpolate(y, size=x, mode="nearest")
+        return y
+
+input_0 = Variable(torch.randn(1, 3, 20, 20))
+input_1 = Variable(torch.randn(1, 3, 10, 10))
```

Comment:
Please use smaller but inputs which still are representative. To cover dimensions confusion, try to use different scales and different sizes.